### PR TITLE
[DOC] Uploaded package file structure

### PR DESCRIPTION
**Pull request by nabilbendafi**
Doesn't fixes any code. _"Just"_ documentation.

## Description of the Change

Only documentation

## Checklist

- [ ] ~unit-test added (if change is algorithm)~
- [ ] ~functional test added/updated (if change is functional)~
- [ ] ~man page updated (if applicable)~
- [ ] ~bash completion updated (if applicable)~
- [x] documentation updated
- [x] author name in `AUTHORS`

### CODE REVIEW

File: files/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+ex:
+
+ sha256sum 476e0**cdac6bc757dd2b78bacc13253**23b09c45ecb41d4562deec2a1c7c148405 **my-package_1.2.3_all.deb**
+
+```
+ 0cdac6bc757dd2b78bacc13253_my-package_1.2.3_all.deb
+```
+
+### MD5
```
Comment (smira) [inline files/README.md]: this is actually legacy layout, problem was that it leads to real-world collisions when two first octets and filename are identical. 

aptly still supports it, but only for 'reading' from the pool, it never puts files this way for new package files

Comment (nabilbendafi) [reply files/README.md]: There were a note about MD5 being _legacy_ in the [README.md](https://github.com/aptly-dev/aptly/blob/1c912e7c0b2d8d80275afc637059dd9fac1301ba/files/README.md) that I have updated.
Let me know if it's enough

Comment (smira) [reply files/README.md]: thanks, looks good!

**Comment by smira** [comment]
Despite what GitHub says, this PR was merged, thanks!

**Pull request closed without merging**